docs/readmes/orc8r/deploy_build.md

```diff
@@ -80,6 +80,25 @@ COMPOSE_PROJECT_NAME=magmalte ${PUBLISH} -r ${REGISTRY} -i magmalte -v ${MAGMA_T
 
 ## Build and publish Helm charts
```

Comment:
Add a note below the H2 double-clarifying that you only choose *one* of the subsequent options